test/RibbonOptionsVault.js

```diff
@@ -1273,15 +1354,27 @@ function behavesLikeRibbonOptionsVault(params) {
 
     describe("#rollToNextOption", () => {
       time.revertToSnapshotAfterEach(async function () {
-        this.expectedMintAmount = BigNumber.from("90000000");
+        this.expectedMintAmount = this.isPut
+          ? this.asset === WETH_ADDRESS
```

Comment:
mint amount a bit differen with puts ([gamma docs](https://github.com/ribbon-finance/structured-products/blob/master/contracts/adapters/GammaAdapter.sol#L439))

Reply:
Need to double check this tbh. Worried that there will be a decimal place screw-up

Reply:
I just printed the mint amount when running the contract methods and used this as expected (again, not as straightforwards as 1-1 depositamount-otokens with puts